by-language/java-quarkus-panache/src/main/java/io/crate/MyEntity.java

```diff
@@ -0,0 +1,88 @@
+package io.crate;
+
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.PrePersist;
+import jakarta.transaction.Transactional;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.*;
```

Comment:
Could you please replace `*` with concrete imports?

Reply:
done